### PR TITLE
DVE-2017-0008: Cloudflare omits SOA from NOERROR/NODATA response to DS query

### PR DESCRIPTION
**Pull request by fanf2**

### CODE REVIEW

**Comment by jelu** [comment]
Is this a problem for all undelegated Cloudflare hosted domains or just Nominet?

I think the title needs to be update to reflect that it is Cloudflare DNS.

File: 2017/DVE-2017-0008.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# DVE-2017-0008: missing SOA in NOERROR/NODATA response to DS query
```
Comment (oerdnj) [inline 2017/DVE-2017-0008.md]: Also the DVE name should list the culprit

**Comment by fanf2** [comment]
I don't know of any other insecure delegations from domains hosted on Cloudflare - I would guess that it's relatively rare.

I'll add some clarifications.

**Comment by jelu** [comment]
@DNS-OARC/dve-maintainers Time for someone else to review, go go!

**Review by oerdnj — APPROVED** [review]

**Comment by FiloSottile** [comment]
Oh, hey, yeah, that would be my fault 😁

(Passed it on to the team!)

**Pull request merged**